src/LiLiQ-Rplus-1.1.xml

```diff
@@ -0,0 +1,258 @@
+                 réalisé par un licencié, autre que le logiciel
+                 ou un logiciel modifié, qui produit ou
+                 reproduit la totalité ou une partie importante
+                 du logiciel;
+            <br/>« logiciel modifié » : toute modification par un
+                 licencié de l'un des fichiers source du
+                 logiciel ou encore tout nouveau fichier source
+                 qui incorpore le logiciel ou une partie
+                 importante de ce dernier.
+          </p>
+        </li>
+        <li>
+          <b>3.</b>
+          <p>Licence de droit d'auteur
+            <br/>Sous réserve des termes de la licence, le concédant
+                 accorde au licencié une licence non exclusive
+                 et libre de redevances lui permettant
+                 d’exercer les droits suivants sur le
+                 logiciel :
+          </p>
+          <p>1 Produire ou reproduire la totalité ou une partie importante;
+            <br/>2 Exécuter ou représenter la totalité ou une partie
+                 importante en public;
+            <br/>3 Publier la totalité ou une partie importante.
+          </p>
+          <p>Cette licence est accordée sans limite territoriale et
+             sans limite de temps.</p>
+          <p>L'exercice complet de ces droits est sujet à la
+             distribution par le concédant du code source du
+             logiciel, lequel doit être sous une forme permettant
+             d'y apporter des modifications. Le concédant peut
+             aussi distribuer le logiciel accompagné d'une offre de
+             distribuer le code source du logiciel, sans frais
+             supplémentaires, autres que ceux raisonnables afin de
+             permettre la livraison du code source. Cette offre
+             doit être valide pendant une durée raisonnable.</p>
+        </li>
+        <li>
+          <b>4.</b>
+          <p>Distribution
+            <br/>Le licencié peut distribuer des copies du logiciel,
+                 d'un logiciel modifié ou dérivé, sous réserve
+                 de respecter les conditions suivantes :
+          </p>
+          <p>1 Le logiciel doit être accompagné d'un exemplaire de
+             cette licence;
+            <br/>2 Si le logiciel a été modifié, le licencié doit en
+                 faire la mention, de préférence dans chacun
+                 des fichiers modifiés dont la nature permet
+                 une telle mention;
+            <br/>3 Les étiquettes ou mentions faisant état des
+                 droits d'auteur, des marques de commerce, des
+                 garanties ou de la paternité concernant le
+                 logiciel ne doivent pas être modifiées ou
+                 supprimées, à moins que ces étiquettes ou
+                 mentions ne soient inapplicables à un logiciel
+                 modifié ou dérivé donné.
+          </p>
+        </li>
+        <li>
+          <b>4.1.</b>
```

Comment:
Yep we've been trying to do this for all licenses. Anything with 1.1. and 1.2. gets two &lt;list&gt; levels, and could be more if it's 1.a.i.A., 1.a.i.B., 1.a.ii.A., etc.